prolog.cl

```diff
@@ -841,6 +842,747 @@
+      "Create a virtual warehouse for every 250 Analysts. Monitor to determine how many of these virtual warehouses are being utilized at capacity."
+      "Create a size XL virtual warehouse to support all the dashboard queries. Monitor query runtimes to determine whether the virtual warehouse should be resized."
+      )
+)
+
+
+(multi "Which Snowflake objects does the Snowflake Kafka connector use? (Choose three.)"
+      6
+      '("Pipe"
+      "Internal table stage"
+      "Internal named stage")
+      
+      '("Serverless task"
+      "Internal user stage"
+      "Storage integration"
+      )
+)
+
+  (multi "You are the owner of a table T1 which is in schema S1. The schema is in database D1. In order to grant read-only permissions of this table to a newly created role R1, you will need to…(select all that apply)"
+         6
+        '("Grant ‘USAGE’ on database D1"
+          "Grant ‘USAGE’ on schema S1"
+          "Grant ‘SELECT’ on table T1")
+        
+         '("Grant ‘SELECT’ on database D1"
+           "Grant ‘SELECT’ on schema S1"
+           "Grant ‘USAGE’ on table T1"
+           )
+)
+
+
+   (multi "Which are characteristics of Snowpipe"
+         6
+        '("Load history is stored for 14 days."
+        "Load history must be requested from Snowflake via a REST endpoint, SQL table function, or ACCOUNT_USAGE view."
+        "Loads are combined or split into a single or multiple transactions based on the number and size of the rows in each data file"
+        "Uses Snowflake-supplied compute resources.
+
+")
+         '("Load history is stored for 64 days."
+           "Load history is available upon completion of the COPY statement as the statement output."
+           "Loads are always performed in a single transaction."
+           "Requires a user-specified warehouse to execute COPY statements."
+           )
+)
+
+   (multi "Which are characteristics of batch data loads"
+         6
+
+      '("Load history is stored for 64 days."
+           "Load history is available upon completion of the COPY statement as the statement output."
+           "Loads are always performed in a single transaction."
+           "Requires a user-specified warehouse to execute COPY statements."
+           )
+        '("Load history is stored for 14 days."
+        "Load history must be requested from Snowflake via a REST endpoint, SQL table function, or ACCOUNT_USAGE view."
+        "Loads are combined or split into a single or multiple transactions based on the number and size of the rows in each data file"
+        "Uses Snowflake-supplied compute resources."
+        )
+
+)
+   (fact "Clustering key consists of one or more table columns/expressions, which can be of any data type,"
+         '("except" "including")
+         " GEOGRAPHY, VARIANT, OBJECT, or ARRAY")
+
+
+   (fact "Clustering keys"
+         '("cannot" can"")
```

Comment:
This is a typo.  Also, you should consider using the `cannot` variable defined on line 223 (<https://github.com/jhunt/snowpro-examiner/pull/2/files#diff-6d7433fe50c589cbeeb1ffc2db9dcf4f4a0798ae9897462d16314e5c37b2233bL223>)